pkg/tools/opa/policies.go

```diff
@@ -70,6 +70,7 @@ func PolicyFromFile(p string) (*AuthorizationPolicy, error) {
 		}
 	}
 	return &AuthorizationPolicy{
+		name:         filepath.Base(p),
```

Comment:
We might need to include full path, because we can have conflict by names. 

For example

/a/b/policy.rego
/b/c/policy.rego

So, I think name for policies from file should be full path.

Reply:
Fixed